spec/models/miq_worker_spec.rb

```diff
@@ -206,6 +206,48 @@ def check_has_required_role(worker_role_names, expected_result)
     end
   end
 
+  describe ".config_settings_path" do
+    let(:capu_worker) do
+      ManageIQ::Providers::Amazon::CloudManager::MetricsCollectorWorker
+    end
+
+    it "include parent entries" do
+      expect(capu_worker.config_settings_path).to eq(
+        %i(workers worker_base queue_worker_base ems_metrics_collector_worker ems_metrics_collector_worker_amazon)
+      )
+    end
+
+    it "works for high level entries" do
+      expect(MiqEmsMetricsCollectorWorker.config_settings_path).to eq(
+        %i(workers worker_base queue_worker_base ems_metrics_collector_worker)
+      )
+    end
+  end
+
+  describe ".worker_settings" do
```

Comment:
We already have a context for this [here](https://github.com/ManageIQ/manageiq/blob/5815f1604e41e79042fb535748cc06bdf8dd681b/spec/models/miq_worker_spec.rb#L197)...granted it's nested, but can we reuse that.  Doesn't that already test what's here?